readthedocs/doc_builder/backends/sphinx.py

```diff
@@ -273,22 +266,34 @@ def build(self):
             *self.sphinx_parallel_arg(),
         ]
         if self.config.sphinx.fail_on_warning:
-            build_command.extend(['-W', '--keep-going'])
-        build_command.extend([
-            '-b',
-            self.sphinx_builder,
-            '-d',
-            '_build/doctrees',
-            '-D',
-            'language={lang}'.format(lang=project.language),
-            '.',
-            self.sphinx_build_dir,
-        ])
+            build_command.extend(["-W", "--keep-going"])
+        build_command.extend(
+            [
+                "-b",
```

Comment:
We should use long-form commands for all these, so it's clearer in the output I think?

Reply:
Ugh, of course.. 🙃